examples/selection_tools/unit_circle.py

```diff
@@ -0,0 +1,114 @@
+"""
+Unit circle
+===========
+
+Example with linear selectors on a sine and cosine function that demonstrates the unit circle.
+
+This shows how fastplotlib supports bidirectional events, drag the linear selector on the sine
+or cosine function and they will both move together.
+
+Click on the sine or cosine function to set the colormap transform to illustrate the sine or
+cosine function output values on the unit circle.
+"""
+
+# test_example = false
+# sphinx_gallery_pygfx_docs = 'screenshot'
```

Comment:
could make this animate if you do something like the cov matrix example where you have a function to +1

Reply:
This is very short, the user should just try it out